### ISSUE TITLE
Banner show several times within the same frame

### ISSUE THREAD

**Issue by 1n3JgKl9pQ6cUMrW** [admob-plus/admob-plus#260]
### Is this a bug report?

It could be.

### Environment

- Several Android devices (tablets, phones)
- Several Android versions (version 5, up to version 10)
- Cordova CLI 10
- Cordova Android 9
- Latest release of the plug-in (version 1.0)

### Steps to Reproduce

1. Set-up a test banner, as described [here](https://admob-plus.github.io/docs/cordova/ads/banner#usage).
2. Open the app
3. The banner is loaded once, but "flashes" just after beginning and shows two ads within a second (sometimes three).

### Expected Behavior

Fire up the banner, show one banner.

### Actual Behavior

The banner is fired once, and the listener on `ad load` is also fired one - but I do see two (sometimes three) banners placed in the same frame.

This occurs with both test-ads and production-ads and under several versions of the plug-in (both alpha, bèta and release versions).

I didn't do anything on the AdMob-site (with refresh duration, etc...).

_Once the banner is "stable" (e.g. the second banners replaces the first once in the same frame) everything is okay._

### Reproducible Demo

**I can create a simple repo for it, for know I've made a screencapture.**

1. The app is started
2. The banner is fired ands shows a "first" banner
3. Within a second, another one is showing (replacing the first)
4. I close the app
5. I repeat step 1 / 3

![capture](https://user-images.githubusercontent.com/15701574/112015859-7d2f3600-8b2c-11eb-84ce-2eeae0a56c26.gif)

**Comment by ratson** [comment]
@1n3JgKl9pQ6cUMrW Did you keep `await banner.hide()`?
It is there for demo purpose.

**Comment by 1n3JgKl9pQ6cUMrW** [comment]
Thank you for the reply.

I've been debugging this thing for almost a week now, created three test-apps, read all the source-code line by line, implemented my own event-listener-debug-program, etc... :-)

---

No I did not use the `banner.hide()`, only the code below;

---

```
let banner;

document.addEventListener( 'deviceready', async () => {

  await admob.start();

  banner = new admob.BannerAd( {
    adUnitId: 'ca-app-pub-xxx/yyy',
  } );

  await banner.show();

}, false);
```

And I also tried the simpler version of this code;

```
var $admob_banner;

document.addEventListener( 'deviceready', function() {

  admob.start();

  $admob_banner = new admob.BannerAd( {
    adUnitId: 'ca-app-pub-xxx/yyy',
  } );

$admob_banner.show();

}, false );
```

I tried calling this event from `deviceready`, `on click`, with and without a `timed delay`, etc...

**Tomorrow I can set-up a simple repo for debugging purposes.**

**Comment by ratson** [comment]
@1n3JgKl9pQ6cUMrW I am thinking if the flash is just the SDK itself reloading the ads.
You may want to check https://github.com/googleads/googleads-mobile-android-examples/tree/master/java/admob/BannerExample to see if it is the default behavior.

**Comment by 1n3JgKl9pQ6cUMrW** [comment]
Yeah, I am not sure this is a `bug` or `feature`.

It both happens using `test ID's` and `production ID's` (in several apps and set-ups).

---

Indeed, it looks like if the SDK if "resetting" itself, because I only call the plug-in once and the `on.load event` is also fired once.

But it's weird, I've never seen this before.

I used to use `cordova-admob-pro` from `floatinghotpot` for years, but after a long struggle I decided to migrate my apps to your plug-in.

_I've released two updated versions of my apps now and they both have the 'flickering issue'._

---

**I will investigate it again, tomorrow by setting up a brand-new, clean repo.**

**Comment by 1n3JgKl9pQ6cUMrW** [comment]
After days of debugging, testing and reading all the source-code I can not `solve` this issue.

Most of the times I see two, sometimes even three, banners within five seconds.

This causes a "flickering" user experience, which is annoying at best.

---

But I was able to `hide` the issue by using the `Cordova Splash Screen Plugin` at https://github.com/apache/cordova-plugin-splashscreen

I did set an extra long timeout for the splash screen, and while the flickering still occurs, it's most of the time not visible anymore due to the splash screen in front of it.
